pkg/component/nginxingress/nginxingress.go

```diff
@@ -647,6 +657,26 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 
 		kubernetesutils.SetAlwaysAllowEviction(podDisruptionBudget, n.values.KubernetesVersion)
 
+		destinationHost := fmt.Sprintf("%s.%s.svc.%s", serviceController.Name, serviceController.Namespace, gardencorev1beta1.DefaultDomain)
```

Comment:
Nit: You could use https://github.com/gardener/gardener/blob/c29cc0fe30c09b980cb81d4a1caea6b7bccad244/pkg/utils/kubernetes/service.go#L24-L26 instead

Reply:
Nice catch. Addressed with https://github.com/gardener/gardener/pull/9038/commits/e623c777f0a6fa66b208296f382d5700906b4ee6.